xbmc/cores/AudioEngine/Sinks/pipewire/AESinkPipewire.cpp

```diff
@@ -254,18 +254,25 @@ std::unique_ptr<PIPEWIRE::CPipewire> pipewire;
 
 bool CAESinkPipewire::Register()
 {
-  pipewire = PIPEWIRE::CPipewire::Create();
-  if (!pipewire)
-    return false;
+  try
+  {
+    pipewire = PIPEWIRE::CPipewire::Create();
```

Comment:
We already have try/catch blocks in `PIPEWIRE::CPipewire::Create` It would be nice if we could just catch the exception in there.

Reply:
Moved the exception handling to `CPipewire::Create`.